server/routes/induction/README.md

```diff
@@ -0,0 +1,2 @@
+# Induction
```

Comment:
Added this in the `routes` folder for Inductions so future developers might find it (my thinking is they are more likely to dive straight into the code, rather that reading the top level readme)

Reply:
I'm not sure about this tbh. I think it could start getting messy if we add READMEs all over the place. I think future developers should know they need to check the main README for information.

Reply:
OK